applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.h

```diff
@@ -0,0 +1,405 @@
+    */
+    SymbolicStokes(IndexType NewId, const NodesArrayType& ThisNodes);
+
+    /// Constructor using a geometry object.
+    /**
+    * @param NewId Index of the new element
+    * @param pGeometry Pointer to a geometry object
+    */
+    SymbolicStokes(IndexType NewId, GeometryType::Pointer pGeometry);
+
+    /// Constuctor using geometry and properties.
+    /**
+    * @param NewId Index of the new element
+    * @param pGeometry Pointer to a geometry object
+    * @param pProperties Pointer to the element's properties
+    */
+    SymbolicStokes(IndexType NewId, GeometryType::Pointer pGeometry, Properties::Pointer pProperties);
+
+    /// Destructor.
+    virtual ~SymbolicStokes();
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /// Create a new element of this type
+    /**
+    * Returns a pointer to a new SymbolicStokes element, created using given input.
+    * @param NewId the ID of the new element
+    * @param ThisNodes the nodes of the new element
+    * @param pProperties the properties assigned to the new element
+    * @return a Pointer to the new element
+    */
+    Element::Pointer Create(IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        Properties::Pointer pProperties) const override;
+
+    /// Create a new element of this type using given geometry
+    /**
+    * Returns a pointer to a new FluidElement element, created using given input.
+    * @param NewId the ID of the new element
+    * @param pGeom a pointer to the geomerty to be used to create the element
+    * @param pProperties the properties assigned to the new element
+    * @return a Pointer to the new element
+    */
+    Element::Pointer Create(IndexType NewId,
+        GeometryType::Pointer pGeom,
+        Properties::Pointer pProperties) const override;
+
+    /// Computes the elemental LHS and RHS elemental contributions
+    /**
+     * Given a distance function, computes the time integrated Left Hand Side (LHS)
+     * and Right Hand Side elemental contributions for the two-fluid element.
+     * @param rLeftHandSideMatrix elemental stiffness matrix
+     * @param rRightHandSideVector elemental residual vector
+     * @param rCurrentProcessInfo reference to the current process info
+     */
+    void CalculateLocalSystem(
+        MatrixType &rLeftHandSideMatrix,
+        VectorType &rRightHandSideVector,
+        ProcessInfo &rCurrentProcessInfo) override;
+
+    /// Computes the elemental RHS elemental contribution
+    /**
+     * Given a distance function, computes the time integrated Right Hand Side (RHS)
+     * elemental contribution for the two-fluid element.
+     * @param rRightHandSideVector elemental residual vector
+     * @param rCurrentProcessInfo reference to the current process info
+     */
+    void CalculateRightHandSide(
+        VectorType &rRightHandSideVector,
+        ProcessInfo &rCurrentProcessInfo) override;
```

Comment:
```suggestion
        const ProcessInfo &rCurrentProcessInfo) override;
```

Reply:
Now calling the base class function to compute it.